### DISCUSSION TITLE
When should I call `close` on the ORM instance?

### DISCUSSION THREAD

**Discussion by haoxi911** [mikro-orm/mikro-orm#3866] [Q&A]
I am using Mikro-ORM with sqlite3 in an Electron app, I initialized the database connection when users launch the software, and I am trying to understand if I need to `close` the connection when they close the software.

Per my test, it seems `close` is optional, and sometimes it is slow. I am using sqlite with WAL mode.

**Comment by B4nan** [comment]
I guess you should keep the connection open for the lifetime of your app, and close it when the app closes, but it depends on what you want. I kinda doubt closing an SQLite connection could be slow.
